contrib/devtools/bitcoin-tidy/example_logprintf.cpp

```diff
@@ -20,11 +20,11 @@ static inline void LogPrintf_(const std::string& logging_function, const std::st
 }
 
 #define LogPrintLevel_(category, level, ...) LogPrintf_(__func__, __FILE__, __LINE__, category, level, __VA_ARGS__)
-#define LogPrintf(...) LogPrintLevel_(LogFlags::NONE, Level::None, __VA_ARGS__)
+#define LogInfo(...) LogPrintLevel_(LogFlags::NONE, Level::None, __VA_ARGS__)
```

Comment:
Since the file contains `LogInfo` examples now, we should rename the file as well.